### PR TITLE
More tests2

### PR DESCRIPTION
**Pull request by alexradzin**

### CODE REVIEW

File: src/main/java/com/firebolt/jdbc/connection/FireboltConnection.java [hunk 1]
```diff
@@ -514,87 +528,75 @@ public void setHoldability(int holdability) throws SQLException {
...
 
 	@Override
-	@ExcludeFromJacocoGeneratedReport
 	@NotImplemented
 	public NClob createNClob() throws SQLException {
 		throw new SQLFeatureNotSupportedException();
 	}
 
 	@Override
-	@ExcludeFromJacocoGeneratedReport
```
Comment (ptiurin) [inline src/main/java/com/firebolt/jdbc/connection/FireboltConnection.java:577]: Can you expand on what is this doing and why is it being removed?

File: src/main/java/com/firebolt/jdbc/statement/FireboltStatement.java [hunk 2]
```diff
@@ -433,8 +430,6 @@ public void setFetchDirection(int direction) throws SQLException {
 	}
 
 	@Override
-	@NotImplemented
-	@ExcludeFromJacocoGeneratedReport
 	public int getFetchSize() throws SQLException {
 		return 0;
 	}
```
Comment (ptiurin) [inline src/main/java/com/firebolt/jdbc/statement/FireboltStatement.java:435]: This should stay "not implemented". From Oracle's docs, this seems to be the pagination size so 0 would be misleading. https://docs.oracle.com/cd/E11882_01/java.112/e16548/resltset.htm#JJDBC28621

File: src/main/java/com/firebolt/jdbc/statement/preparedstatement/FireboltPreparedStatement.java
Comment (ptiurin) [inline src/main/java/com/firebolt/jdbc/statement/preparedstatement/FireboltPreparedStatement.java:1]: Why are we removing `	@NotImplemented` in this file? Those methods seem to be not implemented.

**Review by stepansergeevitch — APPROVED** [review]
Always good to have more tests

**Review by ptiurin — COMMENTED** [review]
It's somewhat misleading that PR title is "more tests", since there are also changes to functionality.

File: src/main/java/com/firebolt/jdbc/connection/FireboltConnection.java (hunk 1 above)
Comment (alexradzin) [reply src/main/java/com/firebolt/jdbc/connection/FireboltConnection.java:577]: This annotation excludes the code of the method from the test coverage report. I have added test that validates this method, so the annotation can be removed.

File: src/main/java/com/firebolt/jdbc/statement/FireboltStatement.java (hunk 2 above)
Comment (alexradzin) [reply src/main/java/com/firebolt/jdbc/statement/FireboltStatement.java:436]: restored `@NotImplemented`

**Review by ptiurin — APPROVED** [review]

**Pull request merged**